sdk/openai/azure-ai-openai/CHANGELOG.md

```diff
@@ -1,13 +1,5 @@
 # Release History
 
-## 1.0.0-beta.1 (Unreleased)
+## 1.0.0-beta.1 (2023-05-22)
```

Comment:
should we bump this to `beta.2`?

Reply:
Have we release the first beta?

Reply:
The drop for LinkedIn, I thought. Or that doesn't count for this version count?

Reply:
Yeah. The one we shared with Linkedin is not released to maven but is our private EngSys repository internally. There is still no namespace existing in the maven repo.